charts/tempo-distributed/values.yaml

```
@@ -179,6 +179,8 @@ ingester:
               matchLabels:
                 {{- include "tempo.selectorLabels" (dict "ctx" . "component" "ingester") | nindent 12 }}
             topologyKey: topology.kubernetes.io/zone
+  # -- Override Pod Disruption Budget maxUnavailable with a static value
+  maxUnavailable: null
```

@sy-be I think it would be good if `.Values.ingester.maxUnavailable` could be commented with a static value to let others know about this configuration.

```suggestion
  # maxUnavailable: 1
```

---

Agree, serves as a better example 👍

---

@sy-be Since you commented out `.Values.ingester.maxUnavailable` in the latest commit, you'll need to run `helm-docs` again in order to address the docs error in the failed CI lint test.